src/coreclr/jit/optimizer.cpp

```diff
@@ -9176,6 +9226,30 @@ GenTree* OptBoolsDsc::optIsBoolComp(OptTestInfo* pOptTest)
 //              |  \--*  LCL_VAR   int    V03 arg3
 //              \--*  CNS_INT   int    0
 //
+//      Case 5:     (x != 0 && y != 0) => (x | y) != 0
```

Comment:
Should be `(x != 0 || y != 0)` I'm assuming?

Reply:
yes it should be (x != 0 || y != 0)